apps/wallet/src/ui/app/pages/qredo-connect/components/QredoAccountItem.tsx

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { CheckFill16 } from '@mysten/icons';
+import { formatAddress } from '@mysten/sui.js';
+import cn from 'classnames';
+
+import { type Wallet } from '_src/shared/qredo-api';
+import { BadgeLabel } from '_src/ui/app/components/BadgeLabel';
+import { Text } from '_src/ui/app/shared/text';
+
+export type QredoAccountItemProps = Wallet & {
+    selected: boolean;
+    onClick: () => void;
+};
+
+export function QredoAccountItem({
+    selected,
+    address,
+    onClick,
+    labels,
+}: QredoAccountItemProps) {
+    return (
+        <div
+            className="flex flex-row items-center flex-nowrap group gap-3 py-4 cursor-pointer"
+            onClick={onClick}
+        >
+            <CheckFill16
+                className={cn('w-4 h-4 text-gray-45 flex-shrink-0', {
+                    'text-success': selected,
+                    'group-hover:text-gray-60': !selected,
+                })}
+            />
+            <div className="flex flex-col flex-nowrap gap-2">
```

Comment:
this is more of a general question, but I see we're using `flex-nowrap` and `flex-1` in a lot of places. Just wanted to check if a lot of these are actually needed or not 😄

Reply:
good question, I think in this case it's needed